src/order-history/service.js

```diff
@@ -22,9 +22,10 @@ export function configureApiService(newConfig, newApiClient) {
   apiClient = newApiClient;
 }
 
-export async function getOrders(page = 1, pageSize = 20) {
+export async function getOrders(username, page = 1, pageSize = 20) {
```

Comment:
@douglashall: Did you also look into paging with the API to see if this is set up correctly?

Reply:
I'm looking into that now as a separate concern from this.